.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Run Unit-Tests
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+
+jobs:
+    unit-tests:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: read
+        steps:
+            - name: Checkout Repo
+              uses: actions/checkout@v4
+            - name: Set up JDK 17
+              uses: actions/setup-java@v4
+              with:
+                  distribution: 'oracle'
+                  java-version: '17'
+            - name: Setup Gradle
+              uses: gradle/actions/setup-gradle@v3
+            - name: Run test suites
```

Comment:
Add newline here as well

Reply:
you want a new line at the end right ??

Reply:
Yes

Reply:
added blank lines at end for both the files